docs/site_organization/multi_file_content_management.md

```diff
@@ -1,4 +1,4 @@
-# Multi-file content management
+# Managing content in bulk
```

Comment:
I have mixed feelings about naming it in bulk as on the UI we have Multi-file, may be confusing for new users, I guess. But this is just my suggestion :)